torch/csrc/jit/passes/lower_tuples.cpp

```diff
@@ -21,6 +21,7 @@ std::unordered_set<Symbol> supported_ops = {
     prim::TupleSlice,
     prim::Param,
     prim::Return,
+    aten::format,
```

Comment:
What's the context for this change? Looks similar to https://github.com/pytorch/pytorch/pull/41548.

Reply:
So an exception msg from torch.nn.functional with string.format appears in the graph as prim::RaiseException node with a tuple which contains aten::format. So I'm adding this op type to the allowed list to enable lower_tuple pass.
This unblocks scripting a number of tests on pytorch_onnx_onnxruntime.